blog/_posts/2017-12-11-Thanks-for-diversifying-tech-with-us.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Thanks for diversifying tech with us!
+layout: post
+created_at: Mon December 11 2017
+permalink: blog/2017-12-11-Thanks-for-diversifying-tech-with-us
+current: blog
+author: Inês Coelho
+twitter: ines_opcoelho
+categories:
+- news
+---
+
+For the past five years, every (north hemisphere) summer, RGSoC has been offering the opportunity for women (transgender and cisgender) and non-binary people from all over the world to work on an Open Source project. This wouldn't be possible without the help and love of our wonderful community. 
```

Comment:
I'd probably remove “women (transgender and cisgender)” and instead just write “all women”?

Reply:
Done! Although the original expression was yours™, copy/pasted from our slack conversation 😂